.gitignore

```diff
@@ -19,4 +19,4 @@ build_*
 
 # Python cache
 __pycache__
-*.pyc
+*.pyc
```

Comment:
May I understand why this file was touched?

Reply:
Can you just add a new line at the end of the file so that the ci tests pass and this does not show up. It seems you have removed the end of line marker on the file.